RecoEgamma/EgammaTools/plugins/EGExtraInfoModifierFromDB.cc

```diff
@@ -364,118 +426,128 @@ namespace {
-    clusterDEtaToSeed[iclus] = pclus->eta() - theseed->eta();
-    
-    // find cluster with max dR
-    const auto the_dr = reco::deltaR(*pclus, *theseed);
-    if(the_dr > maxDR) {
-      maxDR = the_dr;
-      clusterMaxDR = maxDR;
-      clusterMaxDRDPhi = clusterDPhiToSeed[iclus];
-      clusterMaxDRDEta = clusterDEtaToSeed[iclus];
-      clusterMaxDRRawEnergy = clusterRawEnergy[iclus];
-    }      
-    ++iclus;
+    clusterRawEnergy[i_cluster] = pclus->energy();
+    clusterDPhiToSeed[i_cluster] = reco::deltaPhi(pclus->phi(),theseed->phi());
+    clusterDEtaToSeed[i_cluster] = pclus->eta() - theseed->eta();    
+    i_cluster++;
   }
   
-  eval[16] = clusterMaxDR;
-  eval[17] = clusterMaxDRDPhi;
-  eval[18] = clusterMaxDRDEta;
-  eval[19] = clusterMaxDRRawEnergy/raw_energy;
-  eval[20] = clusterRawEnergy[0]/raw_energy;
-  eval[21] = clusterRawEnergy[1]/raw_energy;
-  eval[22] = clusterRawEnergy[2]/raw_energy;
-  eval[23] = clusterDPhiToSeed[0];
-  eval[24] = clusterDPhiToSeed[1];
-  eval[25] = clusterDPhiToSeed[2];
-  eval[26] = clusterDEtaToSeed[0];
-  eval[27] = clusterDEtaToSeed[1];
-  eval[28] = clusterDEtaToSeed[2];
+  eval[25] = clusterRawEnergy[0]/raw_energy;
+  eval[26] = clusterRawEnergy[1]/raw_energy;
+  eval[27] = clusterRawEnergy[2]/raw_energy;
+  eval[28] = clusterDPhiToSeed[0];
+  eval[29] = clusterDPhiToSeed[1];
+  eval[30] = clusterDPhiToSeed[2];
+  eval[31] = clusterDEtaToSeed[0];
+  eval[32] = clusterDEtaToSeed[1];
+  eval[33] = clusterDEtaToSeed[2];
   
   // calculate coordinate variables
-  const bool iseb = ele.isEB();  
   float dummy;
   int iPhi;
   int iEta;
   float cryPhi;
   float cryEta;
   EcalClusterLocal _ecalLocal;
-  if (ele.isEB()) 
+  if (iseb) 
     _ecalLocal.localCoordsEB(*theseed, *iSetup_, cryEta, cryPhi, iEta, iPhi, dummy, dummy);
   else 
     _ecalLocal.localCoordsEE(*theseed, *iSetup_, cryEta, cryPhi, iEta, iPhi, dummy, dummy);
 
   if (iseb) {
-    eval[29] = cryEta;
-    eval[30] = cryPhi;
-    eval[31] = iEta;
-    eval[32] = iPhi;
+    EBDetId ebseedid(theseed->seed());
+    int ieta = ebseedid.ieta();
+    int iphi = ebseedid.iphi();
+    eval[34] = ieta;
+    eval[35] = iphi;
+    int signieta = ieta > 0 ? +1 : -1; /// this is 1*abs(ieta)/ieta in original training
+    eval[36] = (ieta-signieta)%5;
+    eval[37] = (iphi-1)%2;
+    eval[38] = (abs(ieta)<=25)*((ieta-signieta)) + (abs(ieta)>25)*((ieta-26*signieta)%20);  
+    eval[39] = (iphi-1)%20;
   } else {
-    eval[29] = the_sc->preshowerEnergy()/the_sc->rawEnergy();
+    EEDetId eeseedid(theseed->seed());
+    eval[34] = eeseedid.ix();
+    eval[35] = eeseedid.iy();
+    eval[36] = raw_es_energy/raw_energy;
   }
-
+  
   //magic numbers for MINUIT-like transformation of BDT output onto limited range
   //(These should be stored inside the conditions object in the future as well)
-  constexpr double meanlimlow  = 0.2;
-  constexpr double meanlimhigh = 2.0;
+  constexpr double meanlimlow  = -1.0;
```

Comment:
Looks like the comment about saving these magic numbers in the conditions did not serve the purpose.
Better happen next time.